lib/github-provider.js

```diff
@@ -68,9 +71,57 @@ function mapGithubCommitsSince(camp, githubApiUrl, githubAuth) {
     }));
 }
 
+function mapGithubReleaseDate(camp, githubApiUrl, githubAuth) {
+  camp.route(/^\/github\/release-date\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+  cache(function(data, match, sendBadge, request) {
+    const user = match[1];  // eg, microsoft
+    const repo = match[2];  // eg, vscode
+    const format = match[3];
+    const apiUrl = `${githubApiUrl}/repos/${user}/${repo}/releases/latest`;
+    const badgeData = getBadgeData('release date', data);
+    if (badgeData.template === 'social') {
+      badgeData.logo = getLogo('github', data);
+    }
+    githubAuth.request(request, apiUrl, {}, function(err, res, buffer) {
+      if (err != null) {
+        badgeData.text[1] = 'inaccessible';
+        sendBadge(format, badgeData);
+        return;
+      }
+      try {
+        const data = JSON.parse(buffer);
+  
+        if (!data.created_at) throw 'Project has no releases';
+  
+        const releaseDate = moment(data.created_at);
+       
+        // Pretty print release date
+        badgeData.text[1] = releaseDate.calendar(null, {
+          lastDay : '[Yesterday]',
+          sameDay : '[Today]',
+          lastWeek : '[last] dddd',
+          sameElse :  now => releaseDate.year() == now.year() ? 'D MMM' : 'D MMM YYYY'
+        });
+  
+        // Calculate color (older release is lighter/greyer color)
+        const colorB = makeColorB('#007ec6', data ); // blue
+        const baseColor = Color(colorB);
+        const weeksSinceRelease = moment().diff(releaseDate, 'weeks');
+        const factor = Math.max(Math.min(weeksSinceRelease / 25, 0.9), 0.1);
```

Comment:
Please don't ask me about this 😊😊😊....

Reply:
Does the color scheme provided by `age` in `color-formatters` seem reasonable enough to use here, instead?

Reply:
Okay, but that is not lighten or brighten color. Anyways, I am cleaning the existing code with `age` function